homeassistant/components/plugwise/climate.py

```diff
@@ -89,9 +88,7 @@ def target_temperature(self) -> float | None:
     @property
     def hvac_mode(self) -> str:
         """Return HVAC operation ie. heat, cool mode."""
-        if (mode := self.device.get("mode")) is None or mode not in self.hvac_modes:
-            return HVAC_MODE_OFF
-        return mode
+        return self.device["mode"]
```

Comment:
We have to check against the available modes in Home Assistant, to ensure we only return modes known to Home Assistant.

Reply:
Ok, then I propose we return HVAC_MODE_HEAT as the fallback, instead of HVAC_MODE_OFF.